tests/php/src/Cli/Export/ExportActivePlugins.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Reference site export active plugins.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Tests\Cli\Export;
+
+use AmpProject\AmpWP\Tests\Cli\ExportStep;
+
+final class ExportActivePlugins implements ExportStep {
+
+	/**
+	 * List of plugin slugs to exclude.
+	 *
+	 * @var string[]
+	 */
+	const EXCLUDED_PLUGINS = [
+		'airplane-mode',
+		'amp',
+		'query-monitor',
+		'redis-cache',
+		'wp-rocket',
+	];
+
+	/**
+	 * Process the export step.
+	 *
+	 * @param ExportResult $export_result Export result to adapt.
+	 *
+	 * @return ExportResult Adapted export result.
+	 */
+	public function process( ExportResult $export_result ) {
+		$active_plugins = $this->get_active_plugins();
+
+		$active_plugins = array_filter(
+			$active_plugins,
+			[ $this, 'skip_excluded_plugins' ]
+		);
+
+		foreach ( $active_plugins as $plugin ) {
+			$export_result->add_step( 'activate_plugin', compact( 'plugin' ) );
+		}
+
+		return $export_result;
+	}
+
+	/**
+	 * Get the list of currently active plugins.
+	 *
+	 * @return string[] Array of currently active plugins.
+	 */
+	private function get_active_plugins() {
+		return array_map( static function ( $plugin ) {
+			$filename = basename( dirname( $plugin ) );
```

Comment:
What about a `$plugin` which is just a file and not containing a directory? Would it be better to use:

```suggestion
			$filename = strtok( $plugin, '/' );
```

Reply:
Yes, will adapt.

Reply:
Resolved via https://github.com/ampproject/amp-wp/pull/5162/commits/806df0c4c5832707bf0d80a59d8494e5ea32feb4